geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/geotools/GeoMesaDataStoreFactory.scala

```diff
@@ -28,6 +28,7 @@ object GeoMesaDataStoreFactory {
   val QueryThreadsParam  = new GeoMesaParam[Integer]("geomesa.query.threads", "The number of threads to use per query", default = Int.box(8), deprecatedKeys = Seq("queryThreads", "accumulo.queryThreads"))
   val QueryTimeoutParam  = new GeoMesaParam[Duration]("geomesa.query.timeout", "The max time a query will be allowed to run before being killed, e.g. '60 seconds'", deprecatedParams = Seq(DeprecatedTimeout, DeprecatedAccumuloTimeout), systemProperty = Some(TimeoutSysParam))
   val LooseBBoxParam     = new GeoMesaParam[java.lang.Boolean]("geomesa.query.loose-bounding-box", "Use loose bounding boxes - queries will be faster but may return extraneous results", default = true, deprecatedKeys = Seq("looseBoundingBox"))
+  val StrictBBoxParam    = new GeoMesaParam[java.lang.Boolean]("geomesa.query.loose-bounding-box", "Use loose bounding boxes - queries will be faster but may return extraneous results", default = false, deprecatedKeys = Seq("looseBoundingBox"))
```

Comment:
Hmm...  is there an error if one tries to see both of the bbox params to true:)?

Reply:
they shouldn't both be used by a single data store - the mixin trait `GeoMesaDataStoreParams` (below) picks one based on a config value

Reply:
I see...  what do the override options look like?  When do I have a choice to change the behavior?

Reply:
it's defined by the extending class, like so: https://github.com/elahrvivaz/geomesa/blob/fcr_index_keyspace/geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/data/CassandraDataStoreFactory.scala#L134-L136